app/views/admin/invites/new.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1 Late Submission Invitation
+
+.container
+  .row
+    .col-md-8
+      = semantic_form_for(@invite,  url: admin_conference_invitation_path, html: { multipart: true }) do |f|
+        = f.input :emails, class: 'form-control', required: true
+        = f.input :content, as: :select, collection: ['Track', 'Booth'], label: "Invite For"
+        = f.input :end_date, as: :string, input_html: { id: 'invitation-end-date', end_date: @conference.end_date }
+
+        %p.text-right
+          = f.submit 'Submit', class: 'btn btn-success'
```

Comment:
How about we make this button show something more descriptive? Like 'Allow late submissions'

Moreover, and referring to another one of my comments, this button could be actually a drop down where the user can select if an email will be sent informing the user about the ability to submit proposals. 

That would also require an email template, and only having the email option available in the dorp down if an email template actually exists. This could be implemented in a different PR if you want to.

Reply:
@differentreality I will add this in the todo doc :smile:. I will first add a custom email for the invitation that we are sending to unregistered users after that I will work on this.